### PR TITLE
adding initial proto file

### PR DESCRIPTION
**Pull request by grubino**
What else should go in here?

### CODE REVIEW

**Comment by recursivelycurious** [comment]
@grubino see comments on #2 - some context would be helpful. One general note; if necessary to facilitate exploration and discussion, I believe its valid to have a project be private initially and open it at an appropriate alpha/beta stage. Not sure if that is needed here, but wanted to mention in case the sparseness of the repo / issues / commit logs is due to playing close to the vest rather than lack of clarity?

**Review by recursivelycurious — CHANGES_REQUESTED** [review]
More information and context please.

**Comment by grubino** [comment]
I will add a README soon @recursivelycurious

**Comment by grubino** [comment]
Added more README stuffs.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Welcome to ñIFTy Messenger!  We here at ñIFT are dedicated to getting your messages to only every person to which you intend them to send.
+
+## Front Matter
+
+This project is currently in development and incomplete.  As a result it will change often.
+
+### Contribution guidelines
+
+Contributors to ñIFTy Messenger should be aware that any and all code to be merged into a release branch is subject to review by ñIFT members.
```
Comment (grubino) [inline README.md:11]: "will be subject to review prior to merging"

**Comment by grubino** [comment]
k, it's been long enough.  Merging...

**Pull request merged**